orion/sites/admin/src/programs/program.styl

```diff
@@ -52,6 +56,22 @@
   & .li-large
     width: 240px
 
+#view-program #list-buttons div
+    display: inline-block
+
 a.add-program
-  display: inline-block
-  margin: spacing2x 0
+    display: inline-block
+    margin: spacing2x 0
+    background-color: colorTurquoise
+    color: white
+    padding: 0.75em;
+    border-radius: 10px;
```

Comment:
keep in mind proper ordering of CSS

display, (padding, margin, border, etc.) then colors

Reply:
bump again!